README.md

```diff
@@ -1,53 +1,121 @@
-# Awesome Hashgraph [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Join the chat at https://hashgraph.com/discord](	https://img.shields.io/discord/373889138199494658.svg)](https://hashgraph.com/discord)
+# Awesome Hashgraph [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Join the chat at https://hashgraph.com/discord]( https://img.shields.io/discord/373889138199494658.svg)](https://hashgraph.com/discord)
 
 A curated list of Hashgraph application code and other resources, inspired by other awesome lists.
 
-If you want to contribute, please read [this](CONTRIBUTING.md).
-
 ## Table of Contents
 
 - [Official Hashgraph Sites](#official-hashgraph-sites)
-- [Open Source](#open-source)
+- [Reference Key](#key)
+- [Hedera Open Source](#hedera-open-source)
+- [Hedera Projects](#hedera-projects)
+- [Swirlds Open Source](#hedera-open-source)
 - [Communities](#communities)
-- [Projects](#projects)
+- [Contributing](#contributing)
 - [License](#license)
 
-## Official Hashgraph Sites
+## [Official Hashgraph Sites](#official-hashgraph-sites)
+
+* [Hedera](https://hedera.com) - the only public network authorized to implement the hashgraph algorithm.
+* [Swirlds](http://www.swirlds.com/) - inventors of the hashgraph consensus algorithm, offering licenses to use it in permissioned environments.
+
+## [Reference Key](#key)
```

Comment:
The reference key is from the bottom of the existing readme. It's mostly an attempt to explain that the Hedera logo = Officially maintained by Hedera before they read the list.

In the future however it could include other logos recognized by the community in the future (e.g. Swirlds maintained projects, etc.)